src/main/kotlin/com/openlattice/organizations/DatasetController.kt

```diff
@@ -183,6 +185,46 @@ class DatasetController : DatasetApi, AuthorizingComponent {
         edms.deleteOrganizationExternalDatabaseColumns(organizationId, mapOf(tableId to columnIds))
     }
 
+    @Timed
+    @PostMapping(path = [ID_PATH + TABLE_NAME_PATH + EXTERNAL_DATABASE_ROW + POLICY_NAME_PATH])
+    override fun createExternalDatabaseRowSecurityPolicy(
+            @PathVariable(ID) organizationId: UUID,
+            @PathVariable(TABLE_NAME) tableName: String,
+            @PathVariable(POLICY_NAME) policyName: String,
+            @RequestBody rowPolicy: RowSecurityPolicy
+    ) {
+        val tableId = getExternalDatabaseObjectId(organizationId, tableName)
+        val aclKey = AclKey(tableId)
+        //ensureOwnerAccess(aclKey)
```

Comment:
How come the permissions checks are commented out?

Reply:
for testing. They'll be uncommented when it is ready for merge

Reply:
Gotcha!